AI/modules/trader/strategies/portfolio_logic.py

```
@@ -14,61 +15,74 @@ def calculate_portfolio_allocation(
-        for model_name, wrapper in model_wrappers.items():
+        for model_name in model_names:
+            wrapper = model_wrappers[model_name]
             try:
-                # 명세서에 따라 각 모델은 자신만의 필요 데이터(Tech, Log_return, Macro)를
-                # Wrapper 내부에서 추출하여 추론합니다.
-                preds_dict = wrapper.predict(df) 
+                # 각 Wrapper 내부에서 필요 데이터를 전처리하고 추론값을 반환
+                preds_dict = wrapper.predict(df, ticker_id=t_id, sector_id=s_id)
+                # preds_dict가 {'TCN': 0.65} 형태라고 가정
                 ticker_signals.update(preds_dict)
             except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)